jobs/pull-kubernetes-e2e-gke.sh

```diff
@@ -1,84 +0,0 @@
-#!/bin/bash
-# Copyright 2016 The Kubernetes Authors.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-set -o errexit
-set -o nounset
-set -o pipefail
-set -o xtrace
-
-readonly testinfra="$(dirname "${0}")/.."
-
-# TODO(fejta): remove this
-if [[ "${PULL_BASE_REF:-}" == "release-1.0" || "${PULL_BASE_REF:-}" == "release-1.1" ]]; then
-  echo "PR GCE job disabled for legacy branches."
-  exit
-fi
-
-export KUBE_GCS_RELEASE_BUCKET=kubernetes-release-pull
-export KUBE_GCS_RELEASE_SUFFIX="/${JOB_NAME}"
-export KUBE_GCS_UPDATE_LATEST=n
-export JENKINS_USE_LOCAL_BINARIES=y
-export KUBE_FASTBUILD=true
-./hack/jenkins/build.sh
-
-readonly version=$(source build/util.sh && echo $(kube::release::semantic_version))
-if [[ -z "${version:-}" ]]; then
-  echo "Could not find build/util.sh, or kube::release::semantic_version failed." >&2
-  exit 1
-fi
-gsutil -m rsync -r "gs://kubernetes-release-pull/ci/${JOB_NAME}/${version}" "gs://kubernetes-release-dev/ci/${version}-pull-gke"
-# Strip off the leading 'v' from the cluster version.
-export CLUSTER_API_VERSION="${version:1}-pull-gke"
-
-export KUBERNETES_PROVIDER="gke"
-export E2E_MIN_STARTUP_PODS="1"
-
-# Flake detection. Individual tests get a second chance to pass.
-export GINKGO_TOLERATE_FLAKES="y"
-export E2E_NAME="e2e-gke-${NODE_NAME}-${EXECUTOR_NUMBER:-0}"
-export GINKGO_PARALLEL="y"
-
-export GINKGO_TEST_ARGS='--ginkgo.skip=\[Slow\]|\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\]'
```

Comment:
There is both a `jobs/pull-kubernetes-e2e.env` and `jobs/pull-kubernets-e2e-gke.env`
The `pull-kubernetes-e2e.env` is shared between all the PR e2e jobs.